assets/scss/layout/_header.scss

```
@@ -1,4 +1,115 @@
-.header {
-  background: #fefae0;
-  font-size: 32px;
+@use 'sass:map';
+
+@include media-breakpoint-up(lg) {
+  .w-lg-30 {
+    width: 30%;
+  }
+  .w-lg-40 {
+    width: 40%;
+  }
+}
+
+.separator-line-top {
+  border-top: 0.8px solid $primary-200;
+}
+
+.custom-nav-link {
+  color: $neutral-700;
+  font-size: $font-size-sm;
+  display: block;
+  padding-top: 0.343rem;
+  padding-bottom: 0.343rem;
+}
+@include media-breakpoint-up(lg) {
+  .custom-nav-link {
+    font-size: $font-size-base;
+    padding-top: map.get($spacers, 1);
+    padding-bottom: map.get($spacers, 1);
+    padding-left: map.get($spacers, 3);
+    padding-right: map.get($spacers, 3);
+  }
+}
+
+.cart-toggle-btn {
+  color: $neutral-700;
+  padding: map.get($spacers, 2);
+  border-width: 0;
+  border-radius: 50%;
+  background-color: transparent;
+  &.active {
+    background-color: $primary-50;
+  }
+}
+@include media-breakpoint-up(lg) {
+  .cart-toggle-btn {
+    padding: map.get($spacers, 3);
+  }
+}
+.cart-badge {
+  font-size: $font-size-xxs;
+  transform: translate(-73.333%, -22.222%);
+}
+@include media-breakpoint-up(lg) {
+  .cart-badge {
+    font-size: $font-size-xs;
+    transform: translate(-50%, -40%);
+  }
+}
+
+.member-menu {
+  right: 0;
+  top: 4rem;
+  min-width: 12.25rem;
+}
+.member-menu-toggle-btn {
+  color: $neutral-700;
+  padding: map.get($spacers, 3);
+  border-width: 0;
+  border-radius: 50%;
+  background-color: $primary-50;
+  &:hover:focus,
+  :active {
+    background-color: $primary-50;
+  }
+  &.collapsed {
+    background-color: transparent;
+    &:hover:focus,
+    :active {
+      background-color: transparent;
+    }
+  }
+}
+.member-menu-link {
+  color: $neutral-700;
+  font-size: $font-size-sm;
+  display: block;
+  padding-top: 0.343rem;
+  padding-bottom: 0.343rem;
+}
+
+.custom-nav-toggle-btn {
+  transition: none;
+  border-width: 0;
+  border-radius: 50%;
+  padding: map.get($spacers, 2);
+  color: $neutral-700;
+  background-color: $primary-50;
+  &:hover:focus,
+  :active {
+    background-color: $primary-50;
+  }
+  &.collapsed {
+    background-color: transparent;
+    &:hover:focus,
+    :active {
+      background-color: transparent;
+    }
+  }
+}
```

```suggestion
.custom-nav-toggle-btn {
  transition: none;
  border-width: 0;
  border-radius: 50%;
  padding: map.get($spacers, 2);
  color: $neutral-700;
  background-color: transparent; //設定透明背景

  &:hover,//移除focus的設定
  &:active {
    background-color: $primary-50;
  }
  //移除collapsed的設定
}
```

1. 原本的背景應該可以先設定成透明。
2. 我覺得focus的效果沒有很好(點擊後背景顏色會持續到下次點擊才消失)，所以先刪除。
3. 修改上面兩點後，發現collapsed的設定好像也沒什麼作用，所以也先刪除。
4. 目前的`&:hover:focus, :active`只有套用到active，所以改成`&:hover,&:active`